<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>171</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>171</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>171</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(96, 214, 169);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>10</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="act">
     <item row="2" column="1">
      <widget class="QLineEdit" name="years_num">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(53, 214, 153);</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="sum_mes">
       <property name="text">
        <string> Сумма кредита</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="clear">
       <property name="styleSheet">
        <string notr="true">background-color: #FF8073;
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: beige;
 padding: 6px;</string>
       </property>
       <property name="text">
        <string>Сброс</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="years_mes">
       <property name="text">
        <string> Лет для выплаты</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="res">
       <property name="styleSheet">
        <string notr="true">background-color: #FFB273;
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: beige;
 padding: 6px;</string>
       </property>
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="sum_num">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(53, 214, 153);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="interest_num">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(53, 214, 153);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="interest_mes">
       <property name="text">
        <string> Стравка в %</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="5">
      <widget class="Line" name="line">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>191</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="visual">
     <item>
      <layout class="QVBoxLayout" name="sum">
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="suma_viplat">
         <property name="styleSheet">
          <string notr="true">background-color:#00AE68;</string>
         </property>
         <property name="text">
          <string>Сумма выплат:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="suma_viplat_num">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(55, 205, 145);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="over">
       <item>
        <widget class="QLabel" name="pereplata">
         <property name="styleSheet">
          <string notr="true">background-color:#00AE68;</string>
         </property>
         <property name="text">
          <string>Переплата:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pereplata_num">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(55, 205, 145);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="month">
       <item>
        <widget class="QLabel" name="suma_viplat_mes">
         <property name="styleSheet">
          <string notr="true">background-color:#00AE68;</string>
         </property>
         <property name="text">
          <string>Сумма выплат в месяц</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="suma_viplat_mes_num">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(55, 205, 145);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
